embedding/csharp/Tizen.Flutter.Embedding/FlutterApplication.cs

```diff
@@ -122,74 +117,71 @@ protected override void OnCreate()
                     dart_entrypoint_argv = entrypointArgs.Handle,
                 };
 
-                Handle = FlutterDesktopRunEngine(ref windowProperties, ref engineProperties);
-                if (Handle.IsInvalid)
-                {
-                    throw new Exception("Could not launch a Flutter application.");
-                }
+                FlutterEngine = FlutterDesktopEngineCreate(ref engineProperties);
```

Comment:
Shouldn't you check if the value is valid as you did in the C++ side?

Reply:
I added check to validate handle.